snap/snapcraft.yaml

```diff
@@ -175,6 +175,12 @@ apps:
     plugs:
       - network
       - network-bind
+  edgexproxy-cmd:
+    adapter: none
+    command: bin/edgexproxy-wrapper.sh
```

Comment:
could you change this to run the edgexproxy command and not the wrapper? The wrapper is currently setup to only run as a daemon so the options will be lost when using the wrapper.

Reply:
will do